internal/cli/finalizer.go

```diff
@@ -0,0 +1,54 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+const timeout = 5
```

Comment:
Please add the unit of the timeout here, `5` by itself doesn't mean anything.
```suggestion
const timeout = 5 * time.Second
```

Reply:
👍